test/app/ResetPassword.spec.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import {expect} from 'chai';
+
+import ResetPassword from '../../src/app/ResetPassword';
+
+describe('<ResetPassword />', function () {
+  beforeEach(function () {
+    this.instance = mount(<ResetPassword />);
```

Comment:
ResetPassword component has a **resetPassword: PropTypes.func.isRequired**

<img width="659" alt="screen shot 2016-09-22 at 5 08 36 pm" src="https://cloud.githubusercontent.com/assets/1401559/18764324/3f09fa44-80e7-11e6-98b5-efd5faf42535.png">

Shouldn't this test fail?